Reshuffle BLE. (#405)
toitware/web-docs: docs/tutorials/index.mdx

</Box>

</Boxes>

## Starter projects
<Boxes>
<Box title="Print date and time" to="starter">
Write a simple Toit program that prints the date and time on your device.
</Box>

<Box title="Weather station" to="starter/weatherstation">

> Using a BME280 print the temperature in regular intervals.

Reviewer: ```suggestion
Use a BME280 to print the temperature at regular intervals.
```

Author: done.
